config/libvirt-setup/roles/libvirt-system/tasks/vm-import.yml

```diff
@@ -0,0 +1,101 @@
+  command: gzip -d /tmp/{{ base_image }}.gz
+  when: base_image_gz.stat.exists == True
+
+- name: Install {{ base_image }} to LVM
+  command: qemu-img convert -O raw /tmp/{{ base_image }} /dev/{{ storage }}/lv_{{ item.key }}_root
+  when: storage is defined
+
+- name: Install {{ base_image }} to FILE
+  command: creates="{{ qemu_img_path }}/{{ item.key }}.img"
+           cp "/tmp/{{ base_image }}" "{{ qemu_img_path }}/{{ item.key }}.img"
+  when: qemu_img_path is defined
+
+- name: Remove {{ base_image }}
+  file: path="/tmp/{{ base_image }}" state=absent
+
+- name: Prepare cloud-config/user-data
+  template: src="templates/cloud-config/user-data" dest="/tmp/vm-{{ item.key }}/user-data"
+
+- name: Prepare cloud-config/meta-data
+  template: src="templates/cloud-config/meta-data" dest="/tmp/vm-{{ item.key }}/meta-data"
+
+- name: Generate cidata cd image for cloud-init
+  command: mkisofs -o /tmp/vm-{{ item.key }}.iso -V cidata -r -J --quiet /tmp/vm-{{ item.key }}/user-data /tmp/vm-{{ item.key }}/meta-data
+
+- name: Generate add admin script
+  template: src="templates/add_account.sh" dest="/tmp/add_account-{{ item.key }}"
+
+- name: Execute add admin script
+  command: virt-customize --run /tmp/add_account-{{ item.key }} --selinux-relabel -a "{{ qemu_img_path }}/{{ item.key }}.img"
+  when: item.value.virt_customize is defined
+
+- name: Disable cloud-config for virt-install
+  set_fact:
+     disk_cdrom: ""
+  when: item.value.virt_customize is defined
+
+- name: Enable cloud-config for virt-install
+  set_fact:
+     disk_cdrom: "--disk path=/tmp/vm-{{ item.key }}.iso,device=cdrom --wait 10"
+  when: item.value.virt_customize is not defined
+
+- name: Enable LVM Install
+  set_fact:
+     disk: "--disk /dev/{{ storage }}/lv_{{ item.key }}_root,bus=virtio"
+  when: storage is defined
+
+- name: Enable FILE Install
+  set_fact:
+     disk: "--disk path={{ qemu_img_path }}/{{ item.key }}.img,format=qcow2,bus=virtio,cache=none"
+  when: qemu_img_path is defined
+
+### Install the VM using the image we downloaded
+- name: Install VM
+  command: virt-install --import --name {{ item.key }} --autostart --network bridge={{ bridge }},model=virtio --ram {{ item.value.memory }} {{ disk }} {{ disk_cdrom }} --os-type=linux --nographics
+
+- name: Remove cloud-init cdrom
+  command: virsh change-media {{ item.key }} hda --eject --config
+  when: item.value.virt_customize is not defined
+
+- name: Start VM
+  virt: name={{ item.key }}
+        state=running
+  when: item.value.virt_customize is not defined
+
+- name: Give host time to boot
+  pause: seconds=20
+  when: item.value.virt_customize is defined
+
+- name: Register hosts ip address
+  shell: arp -na | grep $(virsh dumpxml {{ item.key }} | xmllint --xpath 'string(//mac/@address)' -) | awk '{ print $2}' | sed 's/[\(\)]//g'
+  register: ip_address
+
+- name: Record hosts ip address
+  template: src="templates/hosts" dest="{{ inventory_dir }}/libvirt-hosts"
+  delegate_to: 127.0.0.1
```

Comment:
I *might* request we change this at some point to anything but this because I think some of the atomic-host-tests playbooks do stuff with this address